### PR TITLE
Add google cloud trace to website flask app

### PR DESCRIPTION
**Pull request by shifucun**
Follow the instruction at: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/trace/cloud-trace-demo-app/app/app.py

Need to deploy to staging and verify

### CODE REVIEW

File: server/__init__.py [hunk 1]
```diff
@@ -20,10 +20,31 @@
...
+from opencensus.trace.propagation import google_cloud_format
+from opencensus.trace.samplers import AlwaysOnSampler
+from opencensus.ext.stackdriver import trace_exporter as stackdriver_exporter
+import opencensus.trace.tracer
+
+propagator = google_cloud_format.GoogleCloudFormatPropagator()
+
+
+def createMiddleWare(app, exporter):
+    # Configure a flask middleware that listens for each request and applies automatic tracing.
```
Comment (pradh) [inline server/__init__.py:34]: nit: line width

**Review by pradh — APPROVED** [review]
wow, so much boilerplate for enabling tracing :)

File: server/__init__.py (hunk 1 above)
Comment (shifucun) [reply server/__init__.py]: Done

Comment (shifucun) [reply server/__init__.py]: Not very sure if this will work, will see..

**Review by pradh — APPROVED** [review]

**Pull request merged**